### PR TITLE
v5.2.2

### PR DESCRIPTION
**Pull request by flaviendelangle**
I will sync with master tomorrow

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -3,6 +3,52 @@
...
+
+### Core
+
+- [core] Add test for space key row checkbox selection toggling (#3262) @alexfauquette
+- [core] Increase CI efficiency (#3441) @oliviertassinari
+- [core] Refactor sorting comparator (#3390) @flaviendelangle
+- [core] Update dependency on the core (#3526) @oliviertassinari
+- [core] Update tweet example in release readme (#3481) @DanailH
+
+
```
Comment (m4theushw) [inline CHANGELOG.md:51]:
```suggestion
```

File: CHANGELOG.md [hunk 2]
```diff
@@ -3,6 +3,52 @@
...
+- 🙈 Allow to precisely control which children rows to expand with the new `isGroupExpandedByDefault` prop
+- 🌍 Add Finnish (fiFI) locale (#3485) @kurkle
+- 📚 Documentation improvements
+- 🐞 Bugfixes
+
+### `@mui/x-data-grid@v5.2.2` / `@mui/x-data-grid-pro@v5.2.2`
+
+#### Changes
+
+- [DataGrid] Add new prop `isGroupExpandedByDefault` (#3444) @flaviendelangle
```
Comment (m4theushw) [inline CHANGELOG.md:23]: It's related to the tree data.

```suggestion
- [DataGridPro] Add new prop `isGroupExpandedByDefault` (#3444) @flaviendelangle
```

File: CHANGELOG.md [hunk 3]
```diff
@@ -3,6 +3,52 @@
...
+- [DataGrid] Add `hideable` option to GridColDef (#3433) @alexfauquette
```
Comment (m4theushw) [inline CHANGELOG.md:24]: I'll merge it later.

```suggestion
- [DataGridPro] Fix resizing of right pinned columns (#3502) @m4theushw
- [DataGrid] Add `hideable` option to GridColDef (#3433) @alexfauquette
```

File: CHANGELOG.md [hunk 4]
```diff
@@ -3,6 +3,53 @@
...
 
+## 5.2.2
+
+_Jan 6, 2022_
+
+A big thanks to the 9 contributors who made this release possible. Here are some highlights ✨:
+
+- 🎁 Add `hideable` option to `GridColDef` (#3433) @m4theushw
+- ⚡ Add support for column-based `sortingOrder` with the new `sortingOrder` option in `GridColDef` (#3449) @Quppa
+- ⚡ Allow to initialize the `page` and `pageSize` without controlling them with the `initialState` prop (#3495) @flaviendelangle
```
Comment (DanailH) [inline CHANGELOG.md:14]: Maybe add a different emoji.

Comment (DanailH) [inline CHANGELOG.md:14]: Maybe add a different emoji.

```suggestion
- ✨ Allow to initialize the `page` and `pageSize` without controlling them with the `initialState` prop (#3495) @flaviendelangle
```

File: CHANGELOG.md [hunk 5]
```diff
@@ -3,6 +3,54 @@
...
+- [DataGrid] Add the page and pageSize to the initialState prop (#3495) @flaviendelangle
+- [DataGrid] Avoid re-render when pressing key inside already focused cell (#3484) @m4theushw
+- [DataGrid] Close other actions menus when opening a new one (#3492) @m4theushw
+- [DataGrid] Deprecate `getValue` param from the cell and row params (#3369) @flaviendelangle
+- [DataGrid] Fix value parsing in date input (#3307) @alexfauquette
+- [DataGrid] Fix can't enter 0 on numeric column (#3491) @m4theushw
+- [DataGridPro] Fix resizing of right pinned columns (#3502) @m4theushw
+- [DataGrid] Fix scrolling bug when an action is focused (#3483) @alexfauquette
+- [DataGrid] Remove `line-height` from `GridCell` (#3446) @DanailH
+- [DataGridPro] Add new prop `isGroupExpandedByDefault` (#3444) @flaviendelangle
```
Comment (m4theushw) [inline CHANGELOG.md:35]:
```suggestion
- [DataGrid] Fix scrolling bug when an action is focused (#3483) @alexfauquette
- [DataGrid] Remove `line-height` from `GridCell` (#3446) @DanailH
- [DataGridPro] Fix resizing of right pinned columns (#3502) @m4theushw
- [DataGridPro] Add new prop `isGroupExpandedByDefault` (#3444) @flaviendelangle
```

File: CHANGELOG.md [hunk 6]
```diff
@@ -3,6 +3,54 @@
...
+- [l10n] Add Finnish (fiFI) locale (#3485) @kurkle
+- [l10n] Improve French (frFR) locale (#3494) @Zenoo
+- [l10n] Improve Italian (itIT) locale (#3452) @destegabry
+- [l10n] Improve Vietnamese (viVN) locale (#3493) @hckhanh
+
+### Docs
+
+- [docs] Generate imports dynamically from the packages export list (#3488 @flaviendelangle
```
Comment (m4theushw) [inline CHANGELOG.md:43]:
```suggestion
- [docs] Generate imports dynamically from the packages export list (#3488) @flaviendelangle
```

File: CHANGELOG.md [hunk 7]
```diff
@@ -3,6 +3,54 @@
...
+- [docs] Make demos compatible with `preProcessEditCellProps` (#3453) @m4theushw
+
+### Core
+
+- [core] Add test for space key row checkbox selection toggling (#3262) @alexfauquette
```
Comment (m4theushw) [inline CHANGELOG.md:48]:
```suggestion
- [test] Add test for row checkbox toggling using the Space key (#3262) @alexfauquette
```

**Review by m4theushw — COMMENTED** [review]
Do you want to include your other approved PRs?

File: CHANGELOG.md (hunk 5 above)
Comment (flaviendelangle) [reply CHANGELOG.md:35]: I will add my PR exposing the tree data field as it is approved.
If the rest of the team wants to merge other approved PRs, I'll only release after my meetings :+1:

**Review by DanailH — APPROVED** [review]

**Review by alexfauquette — APPROVED** [review]

**Pull request merged**